### PR TITLE
{tools}[GCCcore/9.3.0] gettext v0.20.1

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/58022c41be9b7235d79eee2a3a3e8b3f for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/2388638027cf9e75f93a62d17d73e5f1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3107.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/f2ccee990d2f26bde51f1e86670d9e76 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/956479762abc9001294b6ba1db9dfe0b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2500.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2de90d6ee4c76b4deb51b947e59e289b for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/6632549bd90bee15173f37d0eb4fc9ef for a full test report.

File: easybuild/easyconfigs/n/ncurses/ncurses-6.2-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    local_common_configopts + "--enable-ext-colors --enable-widec --includedir=%(installdir)s/include/ncursesw/",
+]
+
+local_libs = ["form", "menu", "ncurses", "panel"]
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp",
+                                     "infotocap",
+                                     "ncurses%(version_major)s-config",
+                                     "reset", "tabs", "tic", "toe", "tput",
+                                     "tset"]] +
```
Comment (lexming) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.2-GCCcore-9.3.0.eb]: Very minor change: I would reduce the list of bins to two lines
```suggestion
    'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses%(version_major)s-config",
                                     "reset", "tabs", "tic", "toe", "tput", "tset"]] +
```

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/2e79722dd85fa9e6445a9e7031a7a89e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/82d1ce2552267d1441b5204edb8a0a43 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @akesandgren !

**Pull request merged**